packages/api/package.json

```diff
@@ -43,6 +43,8 @@
     "neverthrow": "^6.0.0",
     "openapi-types": "^12.1.0",
     "remeda": "^1.12.0",
+    "resend": "^2.1.0",
+    "svix": "^1.15.0",
```

Comment:
We need to implement this without introducing a svix dependency.

Reply:
<img width="1193" alt="image" src="https://github.com/dittofeed/dittofeed/assets/62795688/9e9f20d0-0991-4842-ad44-542fe77cb279">

SVIX (Webhooks as a Service) recommends using their library to verify the payloads and also I spoke to Resend's team they said the same thing

Let me know if you still want to proceed with verifying it manually.